tool/gravity/cli/config.go

```diff
@@ -608,6 +630,21 @@ func (i *InstallConfig) updateClusterConfig(resources []storage.UnknownResource)
 	if globalConfig.CloudProvider != "" {
 		i.CloudProvider = globalConfig.CloudProvider
 	}
+	if i.ServiceCIDR != "" && globalConfig.ServiceCIDR == "" {
+		globalConfig.ServiceCIDR = i.ServiceCIDR
+	}
+	if i.PodCIDR != "" && globalConfig.PodCIDR == "" {
+		globalConfig.PodCIDR = i.PodCIDR
+	}
+	globalConfig.PodCIDR, err = validate.NormalizeSubnet(globalConfig.PodCIDR)
```

Comment:
Turn user's `10.244.0.10/16` -> `10.244.0.0/16`. This definitely makes sense for service IP range (since API server will still use the first IP starting from `10.244.0.0` and not `10.244.0.10`) so I duplicated the same for pod IP range.